doc/interface.rst

```diff
@@ -426,6 +426,12 @@ Tool Bar
   The associated pop-up
   allows the color of the displayed residuals to be changed.
 
+:icon:`location` Edit Camera Registration Points
+  Enables editing the |crp|\ s associated with a |gcp|.
```

Comment:
```suggestion
  Toggles editing of |crp|\ s.
  See :doc:`cameracalibration` for details.
```
(Consistency with GCPs, please! Removal of the second half of the first sentence is deliberate; a CRP needn't have an associated GCP. Also, I'd encourage adding a TODO to update the reference.)